core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -53,6 +53,7 @@ import org.apache.kafka.common.{Node, TopicPartition}
 import org.apache.kafka.common.requests.SaslHandshakeResponse
 import org.apache.kafka.common.resource.{Resource => AdminResource}
 import org.apache.kafka.common.acl.{AccessControlEntry, AclBinding}
+import DescribeLogDirsResponse.{LogDirInfo, ReplicaInfo}
```

Comment:
My bad. It is removed now. I will go over the patch again to look for unused import.